alkemio_virtual_contributor_engine/chromadb_utils.py

```
@@ -0,0 +1,136 @@
+    """
+    if not chromadb_client:
+        logger.error("ChromaDB client not initialized")
+        return
+    if not embeddings:
+        logger.error("Embeddings model not initialized")
+        return
+
+    try:
+        chromadb_client.delete_collection(collection_name)
+        logger.info(f"Collection {collection_name} deleted.")
+    except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)